### DISCUSSION TITLE
Keycloak integration with Sunbird lern module

### DISCUSSION THREAD

**Discussion by pritiranjanc** [Sunbird-Ed/Community#342] [General Discussion]
We are trying to run sunbird lms service in local machine and facing the below issues while registering the user. 
1. How to add Casandra Storage Federation Provider to KeyCloak, Currently It is showing only LDAP and Kuberoes.
2. User Information is getting stored in cassandra DB when we create a user, But we are unable to fetch the same User Information from KeyCloak. (Update Password call is failing for keyCloak)

Kindly help us addressing the above issues.

**Comment by Krishnaj20** [comment]
@gandham-santhosh @vinukumar-vs - Pls check this query and respond.

**Reply by gandham-santhosh** [reply]
@pritiranjanc can you please share the documentation you have followed to set up Lern module?
cc: @reshmi-nair @rhwarrier

**Reply by pritirc** [reply]
@gandham-santhosh  
we have followed below link for setting up user org service.
https://lern.sunbird.org/use/developer-installation/user-and-org-service

**Reply by vinukumar-vs** [reply]
@pritiranjanc @pritirc This is related to Keycloak integration which is handled by Sunbird-Lern BB. It would be good to move this discussion thread to [Sunbird-Lern discussions](https://github.com/orgs/Sunbird-lern/discussions). It will be useful for future reference for Lern BB. 
cc: @reshmi-nair @rhwarrier 

@Krishnaj20 @gandham-santhosh
